corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -4,12 +4,12 @@
 hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
     // 'hqwebapp/js/hq.helpers' is a dependency. It needs to be added
     // explicitly when webapps is migrated to requirejs
-    var kissmetrics = hqImport("analytix/js/kissmetrix");
-    var FormplayerFrontend = hqImport("cloudcare/js/formplayer/app");
-    var separator = " to ",
-        dateFormat = "YYYY-MM-DD";
-    var selectDelimiter = "#,#"; // Formplayer also uses this
-    var Const = hqImport("cloudcare/js/form_entry/const"),
+    let kissmetrics = hqImport("analytix/js/kissmetrix"),
+        FormplayerFrontend = hqImport("cloudcare/js/formplayer/app"),
+        separator = " to ",
+        dateFormat = "YYYY-MM-DD",
+        Const = hqImport("cloudcare/js/form_entry/const"),
+        Util = hqImport("cloudcare/js/formplayer/utils/util"),
         Utils = hqImport("cloudcare/js/form_entry/utils"),
```

Comment:
nit: having `Util` and `Utils` in the same namespace is confusing - consider renaming one?

Reply:
The best part is that `util` is in a `utils` folder. Yeah, I'll take a look in another PR.

Reply:
https://github.com/dimagi/commcare-hq/pull/32148